update colors, favicon, font (#8)
0xcatalysis/docs: docs/ssp/faqs.md

---
sidebar_position: 3
---

# FAQs

1. **How does a new restaking protocol integrate into Catalysis?**

> New restaking protocols can integrate with Catalysis through our standardized integration process. The Catalysis team works closely with SSP teams to implement the necessary adapters and interfaces, ensuring seamless compatibility with the Catalysis Security Abstraction Layer.

Reviewer: ```suggestion
New restaking protocols can integrate with Catalysis through our standardized integration process. The Catalysis team works closely with restaking protocol teams to implement the necessary integrations, ensuring seamless compatibility with the Catalysis Security Abstraction Layer.
```

> 2. **Does Catalysis support restaking protocols (SSP) across all chains?**

Reviewer: ```suggestion
2. **Does Catalysis support shared security protocols (SSP) across all chains?**
```

No, it doesn't currently. Catalysis currently supports EVM-based SSPs like Eigenlayer and Symbiotic, as well as Cosmos-based protocols like Babylon and Satlayer. However, Catalysis has a flexible architecture and is VM-agnostic, so it can integrate other restaking protocols on newer chains as the ecosystem evolves.

3. **What are the benefits for restaking protocols to integrate with Catalysis?**

> Restaking protocols benefit from Catalysis through increased demand aggregation, as Catalysis acts as a distribution channel connecting Networks to multiple SSPs. This creates a level playing field where protocols can compete based on features, cost, and performance rather than first-mover advantage.

Reviewer: ```suggestion
Catalysis drives demand aggregation by acting as a distribution channel for Networks seeking economic security and boosts Network participation by making it easier for high-quality Networks to onboard and integrate.
```

> Additionally, Catalysis simplifies Network onboarding, driving ecosystem growth and attracting more high-quality Networks to the integrated SSPs. By aggregating demand from multiple Networks, Catalysis also helps create more stable and predictable liquidity requirements for restaking protocols. This aggregation effect can reduce volatility in staking/unstaking patterns and help SSPs attract more consistent staker participation through improved yield opportunities.

Reviewer: Remove this. Need to make these FAQs as crisp as possible.

> 4. **How does Catalysis ensure fair treatment across different restaking protocols?**

Reviewer: Only 3 points are sufficient
